### DISCUSSION TITLE
How does remove unregistered torrents work?

### DISCUSSION THREAD

**Discussion by 2fst4u** [StuffAnThings/qbit_manage#145] [Q&A]
Consider the following hypothetical:

- I use the cross-seed script to look out for torrents that can be cross seeded.
- I use private trackers so I need to seed for a certain ratio/time
- All my apps run in docker so they are functionally different machines
- I therefore have to use the qbittorrent "copy .torrent files to" function so cross-seed can see the torrents to search for

If I use qbittorrent-manage and enable the "remove unregistered torrents" function, the description says it will remove torrents that are not being cross-seeded. 

Does this mean it deletes all torrent files that aren't being cross seeded? What if a torrent shows up in a few days that can be cross seeded but qbittorrent-manage has already deleted the .torrent file? Cross seed won't be able to see it and will miss the cross seed.

I'm very confused about this particular flag.

Edit: I'm looking for an option to delete any .torrent files in the "copied to" location that aren't actually in qbittorrent at all but I can't see one. Only this one that mentions cross seeding but I want it to be unrelated to cross seeding

**Comment by bakerboy448** [comment]
The wiki now documents `rem_unregistered_filter_completed` (settings table) — this flag restricts `rem_unregistered` to completed torrents only. See: https://github.com/StuffAnThings/qbit_manage/wiki/Config-Setup#settings
